javascript/index.js

```diff
@@ -0,0 +1,27 @@
+// printToDom - reusable code for building html from your javascript file
+// Pass in an element id where you would like to print, then a variable name which holds html to print
+
+const printToDom = (divId, toPrint) => {
+  document.getElementById(divId).innerHTML = toPrint;
+};
+
+// arrayToList - loops through an array that does not contain objects and prints each entry as an li element
+// Pass in your array name, then an id for your target UL or OL element
+
+const arrayToList = (arrayName, listId) => {
+  let listToPrint = '';
+  for (let i = 0; i < arrayName.length; i++){
```

Comment:
Your code is very clean and well commented. I don't really have anything to add.
Just for fun, another way to write this is using the forEach method, for some people it looks easier to read. I really like the for (...) though....hehehe
```
arrayName.forEach(item => listToPrint +=`<li>${item}</li>`)
```